### PR TITLE
Document Tart Isolation

### PR DESCRIPTION
**Pull request by fkorotkov**

### CODE REVIEW

File: docs/guide/persistent-workers.md [hunk 1]
```diff
@@ -108,6 +108,26 @@ By default, a persistent worker spawns all the tasks on the same host machine it
 
 However, using the `isolation` field, a persistent worker can utilize a VM or a container engine to increase the separation between tasks and to unlock the ability to use different operating systems.
 
+#### Tart
+
+To use this isolation type, install the [Tart]([https://www.parallels.com/products/desktop/](https://github.com/cirruslabs/tart)) on the persistent worker's host machine.
```
Comment (edigaryev) [inline docs/guide/persistent-workers.md:113]:
```suggestion
To use this isolation type, install the [Tart]([https://www.parallels.com/products/desktop/](https://github.com/cirruslabs/tart) on the persistent worker's host machine.
```

File: docs/guide/persistent-workers.md [hunk 2]
```diff
@@ -108,6 +108,26 @@ By default, a persistent worker spawns all the tasks on the same host machine it
...
+
+Here's an example of a configuration that will run the task inside of a fresh macOS virtual machine created from the a remote `ghcr.io/cirruslabs/macos-ventura-base:latest` VM image:
```
Comment (edigaryev) [inline docs/guide/persistent-workers.md:115]:
```suggestion
Here's an example of a configuration that will run the task inside of a fresh macOS virtual machine created from a remote `ghcr.io/cirruslabs/macos-ventura-base:latest` VM image:
```

File: docs/guide/persistent-workers.md [hunk 3]
```diff
@@ -108,6 +108,26 @@ By default, a persistent worker spawns all the tasks on the same host machine it
...
+    tart:
+      image: ghcr.io/cirruslabs/macos-ventura-base:latest
+      user: admin
+      password: admin
+
+task:
+  script: system_profiler
+```
+
+Once the VM spins up, persistent worker will connect to the VM's IP-address over SSH using `user` and `password` credentials and run the latest agent version targeted for the `platform`.
```
Comment (edigaryev) [inline docs/guide/persistent-workers.md:129]: The YAML configuration doesn't seem to use `platform` field, yet this paragraph mentions it.

Comment (fkorotkov) [reply docs/guide/persistent-workers.md]: Oops. Copy pasta from Parallels. Removed in 4a0936c51aacd600421b25398f2535edb58dbb12

File: docs/guide/persistent-workers.md [hunk 4]
```diff
@@ -108,6 +108,26 @@ By default, a persistent worker spawns all the tasks on the same host machine it
 
 However, using the `isolation` field, a persistent worker can utilize a VM or a container engine to increase the separation between tasks and to unlock the ability to use different operating systems.
 
+#### Tart
+
+To use this isolation type, install the [Tart]([https://www.parallels.com/products/desktop/](https://github.com/cirruslabs/tart) on the persistent worker's host machine.
```
Comment (edigaryev) [inline docs/guide/persistent-workers.md:113]:
```suggestion
To use this isolation type, install the [Tart](https://github.com/cirruslabs/tart) on the persistent worker's host machine.
```

**Review by edigaryev — APPROVED** [review]

**Pull request merged**